clients/fides-js/src/components/fides.css

```diff
@@ -389,24 +387,32 @@ div#fides-consent-content .fides-modal-body {
 }
 
 div#fides-consent-content .fides-modal-description {
-  margin: 1em 0 1em 0;
+  margin: 8px 0 24px 0;
 }
 
 .fides-banner-button-group {
   display: flex;
-  gap: 10px;
+  gap: 12px;
 }
 
 .fides-modal-button-group {
   display: flex;
   width: 100%;
   flex-direction: row;
-  gap: 10px;
+  gap: 12px;
   margin-inline: var(--fides-overlay-padding);
 }
 
+.fides-tcf-banner-container
+  div#fides-banner
+  div#fides-banner-inner
+  div#fides-button-group
+  .fides-banner-button-group {
+  padding-left: 24px;
+}
+
 /* Responsive overlay */
-@media (max-width: 48em) {
+@media (max-width: 768px) {
```

Comment:
I'd prefer to keep this as `em`—I took this [from chakra](https://chakra-ui.com/docs/styled-system/responsive-styles)